qiskit_machine_learning/algorithms/__init__.py

```diff
@@ -117,4 +117,6 @@
     "PyTorchDiscriminator",
     "QuantumGenerator",
     "QGAN",
+    "EffectiveDimension",
+    "LocalEffectiveDimension",
```

Comment:
These should also be added to the docstring section above so the are linked into the API reference material. Maybe they come under some new section 'Dimensionality Reduction' or something?